packages/web-config-server/src/apiV1/measureData.js

```diff
@@ -236,13 +236,15 @@ export default class extends DataAggregatingRouteHandler {
       return Array.from(otherLinkMeasureKeySet);
     };
 
-    // Config 'measureConfig' only works for report server builder
+    // Config 'measureConfig' only works for report-server
     const { measureConfig, ...mainMeasureOption } = measureOptions[0];
     if (measureConfig) {
       const { [ADD_TO_ALL_KEY]: configForAllKeys, ...restOfConfig } = measureConfig;
-      getOtherMeasureOptionKeys(mainMeasureOption.key).forEach(key => {
-        measureOptions.push({ ...configForAllKeys, key, name: key, ...restOfConfig[key] });
-      });
+      getOtherMeasureOptionKeys(mainMeasureOption.key)
+        .filter(key => !key.includes('_metadata'))
```

Comment:
hacky way to filter keys with `_metadata` while adding `measureOptions` for other report server data.

Reply:
Ahhh I see the issue here.... nice hack! Hopefully we can remove this as more overlays move to the report-server 👍